### ISSUE TITLE
3D printer board as "Mainboard"

### ISSUE THREAD

**Issue by sibero80** [isaac879/Pan-Tilt-Mount#14]
Given that most 3D printer boards work with an ATmega processor.. could it be possible to use one of those boards as the base of this project with minor modifications to the code?

**Comment by dronecz** [comment]
Yes, it should be no problem to drive stepper motors with it. There will probably be small issue with sensing of voltage in battery and with controlling your shutter at camera, but that will depend what 3D printer controller you will use and how good you are with modifications of HW 😉
